source/fundamentals/data-formats/documents.txt

```diff
@@ -45,6 +45,11 @@ access and manipulate the BSON data in documents:
      - Yes, implements ``Map<String, BsonValue>``
      - When you need a type-safe API.
 
+   * - ``JsonObject``
+     - ``org.bson.json``
+     - No
+     - When you need to work with JSON strings.
```

Comment:
I would mention that this is when you "only want to work with JSON strings" since this page also shows how to parse and convert JSON with Document.

Reply:
Thanks for the feedback Chris. I agree that "only want to work with JSON strings" is more accurate and helpful language. The update is made in https://github.com/mongodb/docs-java/pull/57/commits/d818607f965d55cf8de602028b8ef15e1525878d.